kernel/kernel-impl/src/main/java/org/sakaiproject/memory/impl/HazelcastMemoryService.java

```diff
@@ -117,22 +134,37 @@ public Properties getProperties() {
         return p;
     }
 
-    @Override
-    public <K, V, C extends Configuration<K, V>> Cache createCache(String cacheName, C configuration) {
-        return new HazelcastCache(makeHazelcastCache(cacheName, configuration));
-    }
-
-    @Override
-    public Cache getCache(String cacheName) {
-        return new HazelcastCache(makeHazelcastCache(cacheName, null));
-    }
+    /**
+     * Currently limiting the use of Hazelcast for caching, so a service has to have its name in the list or it gets an ehcache.
+     * 
+     * @see org.sakaiproject.memory.api.MemoryService#createCache(java.lang.String, org.sakaiproject.memory.api.Configuration)
+     */
+     @Override
+     public <K, V, C extends Configuration<K, V>>  Cache createCache(String cacheName, C configuration) {
+        if (CLIENT_SERVICES.contains(cacheName)) {
```

Comment:
This is where we check. The map only has the cache name for BaseUserEdit objects.